apps/site/pages/components/molecules/accordion.mdx

```diff
@@ -189,30 +237,7 @@ Besides those attributes, the following props are also supported:
 
 ### Accordion
 
-export const propsAccordion = [
-  {
-    name: 'testId',
-    type: 'string',
-    description:
-      'ID to find this component in testing tools (e.g.: cypress, testing library, and jest).',
-    default: 'fs-accordion',
-  },
-  {
-    name: 'indices',
-    type: 'Iterable<number>',
-    description: 'Indices that indicate which accordion items are opened.',
-    required: true,
-  },
-  {
-    name: 'onChange',
-    type: '(index: number) => void',
-    description:
-      'Function that is triggered when an accordion item is opened/closed.',
-    required: true,
-  },
-]
-
-<PropsSection propsList={propsAccordion} />
+<AccordionPropsSection component="Accordion" />
 
 ### Accordion Item
```

Comment:
Accordion Item didn't return props using the lib, maybe because of it's complexity. So we need to mock.

Reply:
So maybe we should remove it from `getStaticProps` and `AccordionPropsSection`, right?